dev/doc/release-process.md

```diff
@@ -65,8 +65,18 @@
 
 ## Only for the final release of each major version ##
 
-- [ ] Ping `@Zimmi48` to publish a new version on Zenodo.
-  *TODO:* automate this with coqbot.
+- [ ] Ping `@coq/zenodo-maintainers` to publish a new version on Zenodo.
```

Comment:
does zenodo look at this team to decide permissions or does adding someone to the team need to be mirrored on the zenodo site?

Reply:
It needs to be mirrored. I've changed the team description to include:

> (changes to the membership on Zenodo must be kept in sync with this GitHub team manually)